source/blog/2021-12-01-mongodb-odm-2.3.md

```diff
@@ -0,0 +1,126 @@
+---
+title: "New Release: Doctrine MongoDB ODM 2.3 with Attributes, JSON Schema Validation and more"
+authorName: Ion Bazan
+authorEmail: ion.bazan@gmail.com
+permalink: /2021/12/01/mongodb-odm-2.3.html
+---
+
+We have released a new minor version 2.3 of Doctrine MongoDB ODM, the first version
+with support for using PHP 8 Attributes as a new driver for mapping documents
+and several other changes. [See all changes and contributors in the
+Changelog](https://github.com/doctrine/mongodb-odm/releases/tag/2.3.0) on GitHub.
+
+## Attributes Mapping Driver
+
+The following code example shows many of the mappings that are re-using
+the annotation classes for familiarity:
+
+```php
+use Doctrine\ODM\MongoDB\Mapping\Annotations as MongoDB;
+use Doctrine\ODM\MongoDB\Types\Type;
+
+#[MongoDB\Document(repositoryClass: PostRepository::class)]
+class Post
+{
+    #[MongoDB\Id]
+    private string $id;
+
+    #[MongoDB\Field(type: Type::BOOLEAN)]
+    private bool $published = false;
+
+    #[MongoDB\Field(type: Types::COLLECTION)]
+    private array $text = [];
+
+    #[MongoDB\ReferenceOne(targetDocument: User::class)]
+    public $author;
+
+    #[MongoDB\ReferenceMany(targetDocument: Tag::class)]
+    public Collection $tags;
+}
+```
+
+You may want to use [Rector](https://getrector.org/) with `DoctrineSetList::DOCTRINE_ODM_23` set
+to convert all your annotation mappings to attributes in seconds!
+
+## JSON Schema Validation
+
+MongoDB ≥ 3.6 offers the capability to validate documents during
+insertions and updates through a JSON schema associated to the collection. 
+[See MongoDB documentation](https://docs.mongodb.com/manual/core/schema-validation/).
+
+Doctrine MongoDB ODM now provides a way to take advantage of this functionality
+thanks to the new `#[Validation]` mapping.
```

Comment:
Given our "annotation reference" page is still not showing up in the menu I'd link to https://www.doctrine-project.org/projects/doctrine-mongodb-odm/en/latest/reference/annotations-reference.html#validation directly